File: Symfony/src/BGR/Serrano/ProductoBundle/Model/PaqueteMix.php
<?php

namespace BGR\Serrano\ProductoBundle\Model;

use JMS\Serializer\Annotation\Type;

class PaqueteMix
{
	/**
	 * @Type("string")
	 * @var unknown
	 * 
	 * producto_id
	 */
	private $id;
	/**
	 * @Type("integer")
	 * @var unknown
	 */
	private $cantidad;
	
	/**
	 * @Type("ArrayCollection<BGR\Serrano\ProductoBundle\Entity\Paquete>")
	 * @var unknown
	 */
	private $paquetes;
	
	/**
	 * @Type("integer")
	 * @var unknown
	 */
	private $cant_elegida;
	
	
	public function getId() {
		return $this->id;
	}
	public function setId($id) {
		$this->id = $id;
		return $this;
	}
	public function getCantidad() {
		return $this->cantidad;
	}
	public function setCantidad($cantidad) {
		$this->cantidad = $cantidad;
		return $this;
	}
	public function getPaquetes() {
		return $this->paquetes;
	}
	public function setPaquetes($paquetes) {
		$this->paquetes = $paquetes;
		return $this;
	}
	
	/**
	 *
	 * @return the unknown
	 */
	public function getCantElegida() {
		return $this->cant_elegida;
	}
	
	/**
	 *
	 * @param
	 *        	$cant_elegida
	 */
	public function setCantElegida($cant_elegida) {
		$this->cant_elegida = $cant_elegida;
		return $this;
	}
	
	

	/*
	 * {"id":1,
	 * "paquetes":[{"id":"145",
	 * "presentacion_id":"9",
	 * "codigo":"un codigo",
	 * "estado":"DISPONIBLE",
	 * "cantidad":"2",
	 * "unidad":"4"}],
	 * "cantidad":2}
	 * 
	 * 
	 * */
	
	
	
}

File: BGR/BGR-Data/src/main/java/ar/com/bgr/serrano/utils/PaqueteMix.java
package ar.com.bgr.serrano.utils;

import java.util.List;

public class PaqueteMix {

	/**
	 * @Type("string")
	 * @var unknown
	 * 
	 * producto_id
	 */
	private Integer id;
	/**
	 * @Type("integer")
	 * @var unknown
	 */
	private Integer cantidad;
	
	/**
	 * @Type("ArrayCollection<BGR\Serrano\ProductoBundle\Entity\Paquete>")
	 * @var unknown
	 */
	private List<PaqueteFromQuery> paquetes;
	
	/**
	 * @Type("integer")
	 * @var unknown
	 */
	private Integer cant_elegida;

	/**
	 * @return the id
	 */
	public Integer getId() {
		return id;
	}

	/**
	 * @param id the id to set
	 */
	public void setId(Integer id) {
		this.id = id;
	}

	/**
	 * @return the cantidad
	 */
	public Integer getCantidad() {
		return cantidad;
	}

	/**
	 * @param cantidad the cantidad to set
	 */
	public void setCantidad(Integer cantidad) {
		this.cantidad = cantidad;
	}

	/**
	 * @return the paquetes
	 */
	public List<PaqueteFromQuery> getPaquetes() {
		return paquetes;
	}

	/**
	 * @param paquetes the paquetes to set
	 */
	public void setPaquetes(List<PaqueteFromQuery> paquetes) {
		this.paquetes = paquetes;
	}

	/**
	 * @return the cant_elegida
	 */
	public Integer getCant_elegida() {
		return cant_elegida;
	}

	/**
	 * @param cant_elegida the cant_elegida to set
	 */
	public void setCant_elegida(Integer cant_elegida) {
		this.cant_elegida = cant_elegida;
	}
	
	
}
